### DISCUSSION TITLE
telescope indirect connection

### DISCUSSION THREAD

**Discussion by cafo78** [Stellarium/stellarium#1590] [Q&A]
Hi guys, newbi here.
I have a question about telescope connection with stellarium. I have EQMode to control my mount on a mini computer which I control through remote desktop with the main computer where I installed  the latest version of stellarium. Now, I read about the possibility to connect stellarium to a remote computer (indirect mode) but I'm not able to set it up properly. I put the ip address of the mini computer and the tcp port? I tried 3389 which connects, but then? I don't see any possibility to see or control the telescope in stellarium.
Installing stellarium directly on the mini computer instead,  I can connect and control the telescope but the problem is I get very low fps (2/3) because the minicomputer is not so powerfull. That's why I'm trying this solution of using stellarium installed on the main computer...

**Comment by alex-w** [comment]
Indirect connection can be used when the device is connected to a remote computer and the software that drives it can 'talk' to Stellarium over the network; this software can be either one of Stellarium's stand-alone telescope servers, or a third party application.

See details: http://stellarium.sourceforge.net/wiki/index.php/Telescope_Control_%28client-server%29

**Comment by cafo78** [comment]
I downloaded stellarium scope and now it works.. I can command the telescope remotely. 

Thank you for your help Alexander 

Mirco
